src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -1551,6 +1601,114 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 
 		navigationController?.present(alert, animated: true)
 	}
+	
+	private func showSRSFlowAlert(
+		for srsFlowAlert: SRSFlowAlert,
+		isLoading: @escaping CompletionBool
+	) {
+		switch srsFlowAlert {
+		case let .preconditionFailed(srsFlowError):
+			showSRSFlowPreconditionAlert(for: srsFlowError)
+		case let .consent(srsFlowConsent):
+			showSRSFlowConsentAlert(for: srsFlowConsent, isLoading: isLoading)
+		case let .error(error):
+			showSRSFlowErrorAlert(for: error, isLoading: isLoading)
+		}
+	}
+	
+	private func showSRSFlowConsentAlert(
+		for consent: SRSFlowAlert.Consent,
+		isLoading: @escaping CompletionBool
+	) {
+		var alert: UIAlertController!
```

Comment:
Please avoid force casting

Reply:
The controller pointer cannot be nil, because he will instantiated in the next line. It's like:

```swift
var result: String!
switch some {
case .foo: result = "Hello"
case .bar: result = "World"
}

// result cannot be nil
```

As discussed in the chat, we'll keep that here, cause of something like this

```swift
let topBottomContainerViewController = TopBottomContainerViewController(
	topController: exposureSubmissionSymptomsViewController ?? UIViewController(),
	bottomController: footerViewController
)
```
is not so elegant with `?? UIViewController()`